README.md

```diff
@@ -34,8 +34,7 @@ Run unit tests with `npm run test`. Jest will pick up test in files with the `.t
 
 To run integration tests:
  
-1. First install chromedriver `sudo ./scripts/install-chromedriver.sh`
-1. Make sure you also have a compatible version of chrome (110)
+1. Make sure have chrome and its matching chromedriver installed
```

Comment:
Why remove the instruction to run the install-chromedriver script here? It's not being used in the integration test image so presumably it's still there as a local convenience?

Reply:
Yeah I had actually intended to remove that file altogether. Having a script that just randomly pops files into `/usr/bin` didn't seem great.

```
snap install chromium
```

seems to install both the browser and the driver.